src/routes/legislative-process/+page.svelte

```diff
@@ -36,14 +38,86 @@
+						แม้กฎหมายจะมีหลายชื่อเรียก และมีลำดับชั้นที่หลากหลาย
+						แต่กฎหมายแม่บททั้งหมดล้วนเกิดขึ้นจากกระบวนการการออกกฎหมายโดย ‘รัฐสภา’ เป็นสำคัญ
+					</p>
 				</div>
 				<div>
 					<h2 class="fluid-heading-04" id="กฎหมายบังคับใช้ได้อย่างไร">กฎหมายบังคับใช้ได้อย่างไร</h2>
 					<hr />
+					<h3 class="fluid-heading-03 my-6" id="การออกกฎหมายตามรัฐธรรมนูญ 2560">
+						กฎหมายที่ออกโดยรัฐสภา ต้องผ่านกระบวนการ 3 ขั้นตอนสำคัญ
+					</h3>
+					<div class="grid grid-cols-1 md:grid-cols-[1fr_16px_1fr_16px_1fr] gap-3">
+						<ProcessCard imgSrc="images/legislative-process/process-01.png" title="เสนอร่างกฎหมาย">
+							ผู้มีสิทธิเสนอร่างของกฎหมายแต่ละประเภท จะแตกต่างกัน
+						</ProcessCard>
+						<ProcessCardArrow />
+						<ProcessCard
+							imgSrc="images/legislative-process/process-02.png"
+							title="พิจารณาโดยรัฐสภา"
+						>
+							<ul class="list-disc list-inside">
+								<li>สส.</li>
+								<li>สว.</li>
+								<li>สภาร่วม (สส.+ สว.)</li>
+							</ul>
+						</ProcessCard>
+						<ProcessCardArrow />
+						<ProcessCard
+							imgSrc="images/legislative-process/process-03.png"
+							title="กลายเป็นกฎหมายโดยสมบูรณ์"
+						>
+							<ul class="list-disc list-inside">
+								<li>ไม่ขัดต่อรัฐธรรมนูญ</li>
+								<li>พระมหากษัตริย์ทรงลงพระปรมาภิไธย</li>
+							</ul>
+						</ProcessCard>
+					</div>
+					<ol class="list-decimal list-inside body-02 mt-6">
+						<li>
+							<b>การเสนอร่างกฎหมาย</b> เป็นการจัดทำร่างกฎหมายและเสนอเข้าต่อรัฐสภา โดยผู้ที่มีสิทธิเสนอร่างกฎหมายจะแตกต่างกันขึ้นอยู่กับว่าเป็นร่างกฎหมายประเภทใดตามที่รัฐธรรมนูญระบุไว้
+							เช่น ร่าง พ.ร.บ. สามารถเสนอโดยคณะรัฐมนตรี หรือ สส. จำนวนไม่น้อยกว่า 20 คน หรือ ประชาชนจำนวน10,000
+							คน
+						</li>
+						<li>
+							<b>การพิจารณาร่างกฎหมายโดยรัฐสภา</b> ร่างกฎหมายจะถูกพิจารณาโดยสมาชิกสภาผู้เแทนราษฎร (สส.)
+							และสมาชิกวุฒิสภา (สว.) ตามลำดับ การพิจารณาของแต่ละสภาจะมี 3 วาระเท่ากัน แต่มีเงื่อนไขและรายละเอียดในการพิจารณาที่แตกต่างกัน
+							ทั้งนี้ ในการพิจารณาร่างกฎหมายบางประเภท จะใช้การประชุมร่วมกันของทั้งสองสภาในการพิจารณา
+							เช่น การพิจารณา ร่าง พ.ร.ป.
+						</li>
+						<li>
+							<b>ร่างกฎหมายกลายเป็นกฎหมายโดยสมบูรณ์์</b> เมื่อร่างกฎหมายไม่ขัดต่อรัฐธรรมนูญ และพระมหากษัตริย์ทรงลงพระปรมาภิไธย
+							ตามด้วยการประกาศในราชกิจจานุเบกษาให้มีผลบังคับใช้เป็นกฎหมาย
+						</li>
+					</ol>
 				</div>
 				<div>
 					<h2 class="fluid-heading-04" id="กฎหมายในเว็บไซต์นี้">กฎหมายในเว็บไซต์นี้</h2>
 					<hr />
+					<p class="body-02 mt-6">
+						เว็บไซต์ Parliament Watch มุ่งนำเสนอกฎหมายที่ผ่านการพิจารณาโดยรัฐสภา
+						เพื่อแสดงให้เห็นถึงกระบวนการทำงานด้านนิติบัญญัติในฐานะตัวแทนประชาชน
+						รวมถึงสะท้อนให้เห็นถึงความคิดเห็นของสมาชิกรัฐสภาต่อประเด็นต่างๆ ในทางกฎหมาย
+						ซึ่งเป็นเครื่องมือในการกำหนดทิศทางของสังคม ทั้งนี้
+						เว็บไซต์นี้จะไม่ปรากฎกฎหมายที่ถูกตราขึ้นผ่านคณะรัฐมนตรีและไม่ต้องผ่านการพิจารณาโดยรัฐสภา
+						เช่น พระราชกฤษฎีกา กฎกระทรวง และกฎหมายที่ตราโดยองค์กรปกครองส่วนท้องถิ่น
+					</p>
+					<img
```

Comment:
Need a margin between paragraph and image. Please refer to Figma.

![image](https://github.com/wevisdemo/parliament-watch/assets/8860448/a447a68a-3edb-4c01-9b1c-7276ba59b382)